src/panels/config/integrations/integration-panels/zwave_js/zwave_js-custom-param.ts

```diff
@@ -0,0 +1,182 @@
+import { LitElement, html, css, CSSResultGroup, nothing } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import "@material/mwc-button/mwc-button";
+import { mdiCloseCircle } from "@mdi/js";
+import "../../../../../components/ha-textfield";
+import "../../../../../components/ha-select";
+import { HomeAssistant } from "../../../../../types";
+
+@customElement("zwave_js-custom-param")
```

Comment:
```suggestion
@customElement("zwave-js-custom-param")
```

Reply:
The underscore is used in all the other zwave_js components